### PR TITLE
Readme

### LINKED ISSUE #1: Fix Build/Code Coverage
- [x] Configure travis to build this repo 
- [x] Configure Coveralls to build repo and report code coverage

### PR DESCRIPTION
**Pull request by zabawaba99**
#resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Firego
 ---
-[![Build Status](https://travis-ci.org/zabawaba99/firego.svg?branch=master)](https://travis-ci.org/zabawaba99/firego) [![Coverage Status](https://coveralls.io/repos/zabawaba99/firego/badge.svg?branch=master)](https://coveralls.io/r/zabawaba99/firego?branch=master)
+[![Build Status](https://travis-ci.org/fullcage/firego.svg?branch=readme)](https://travis-ci.org/fullcage/firego) [![Coverage Status](https://coveralls.io/repos/fullcage/firego/badge.svg)](https://coveralls.io/r/fullcage/firego)
```
Comment (MrGossett) [inline README.md]: `branch=readme`?

File: .travis.yml [hunk 2]
```diff
@@ -10,4 +10,4 @@ before_install:
   - go get github.com/mattn/goveralls
   - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
 script:
-    - $HOME/gopath/bin/goveralls -service=travis-ci -repotoken=$CODE_WALL_TOKEN
+    - $HOME/gopath/bin/goveralls -service=travis-ci -repotoken=COVERALLS
```
Comment (MrGossett) [inline .travis.yml]: is the token literally `COVERALLS`, or is that supposed to be a var like `$COVERALLS`?

Comment (zabawaba99) [inline .travis.yml]: wow...

File: README.md (hunk 1 above)
Comment (zabawaba99) [inline README.md]: fixed

**Comment by MrGossett** [comment]
:shipit:

**Pull request merged**